### PR TITLE
Fix/soc 6148

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

File: exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import java.lang.annotation.Annotation;
+import org.exoplatform.commons.api.security.ExoCSRFProtection;
+
+
+/**
+ * Contract of this class thats constrains access to the resource method that
+ * use JSR-250 security common annotations. See also https://jsr250.dev.java.net
+ * .
+ *
+ * @author <a href="mailto:andrew00x@gmail.com">Andrey Parfonov</a>
```
Comment (fdrouet) [inline exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java]: please remove the author line

File: exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   public void accept(GenericMethodResource method) throws WebApplicationException
+   {
+      for (Annotation a : method.getMethod().getAnnotations())
+      {
+         Class<?> ac = a.annotationType();
+
+         if (ac == ExoCSRFProtection.class)
+         {
+
+            //Check CSRF token
```
Comment (thomasdelhomenie) [inline exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java]: We can maybe do something like that to get the session (not tested yet):
```
ServletContainerRequest request = (ServletContainerRequest) ApplicationContextImpl.getCurrent().getContainerRequest();
            javax.servlet.http.HttpSession session = ((javax.servlet.http.HttpServletRequest) request.getServletRequest()).getSession();
```

File: exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java (hunk 1 above)
Comment (rdenarie) [reply exo.ws.rest.ext/src/main/java/org/exoplatform/services/rest/ext/method/filter/CsrfAccessFilter.java]: done

**Comment by thomasdelhomenie** [comment]
Not required anymore.

**Pull request closed without merging**